### PR TITLE
Intro 1.1 Polished

### PR DESCRIPTION
**Pull request by goldmermaid**

### CODE REVIEW

**Comment by mli** [comment]
Job d2l-zh/PR-615/1 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-615/

File: chapter_introduction/index.md [hunk 1]
```diff
@@ -33,43 +33,76 @@
 
 ## 日常生活中的机器学习
 
-假设现在的你正和本书的作者们坐在一起，驱车去咖啡店。亚历山大用一部iPhone喊出“Hey Siri”，唤醒了手机的语音识别系统。然后，李沐指挥“去星巴克咖啡店”。手机很快显示了这条指令的转录文本。手机还意识到我们在问路，并启动了地图应用程序来满足我们的请求。地图应用程序在启动后就确定了若干条路线。在每条路线旁边，手机都显示了预计的通行时间。当然我们编造这个故事是为了方便教学，但它表明，在短短几秒钟的时间里，我们与智能手机的日常互动可以涉及几种机器学习模型。
-
-现在，请你从基本原则出发，编写一个程序来响应一个“唤醒词“，比如“Alexa”、“小爱同学”和“Hey Siri”。试着在房间里用一台计算机和一个代码编辑器自己编写代码，如 :numref:`fig_wake_word`中所示。问题很难解决。麦克风每秒钟将收集大约44000个样本。每个样本都是声波振幅的测量值。什么规则可以可靠地从原始音频片段映射到关于该片段是否包含唤醒词的可靠预测$\{\text{是}, \text{否}\}$？如果你卡住了，别担心。我们也不知道如何从头开始编写这样一个程序。这就是我们使用机器学习的原因。
+假设你正和本书的作者们一起，驱车去咖啡店。
+亚历山大拿起一部iPhone，对它说道“Hey Siri”--唤醒了手机的语音识别系统。
+接着，李沐对Siri说道“去星巴克咖啡店”--点亮了语音识别系统的语音转文字功能，并启动了地图应用程序来满足我们的请求。
```
Comment (astonzhang) [inline chapter_introduction/index.md:38]: 说到
“唤醒了手机的语音识别系统“ 这句话没有主语

Comment (astonzhang) [reply chapter_introduction/index.md:38]: ”点亮了语音识别系统的语音转文字功能，并启动了地图应用程序来满足我们的请求“ 也缺了主语

**Comment by mli** [comment]
Job d2l-zh/PR-615/2 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-615/

**Comment by mli** [comment]
Job d2l-zh/PR-615/3 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-615/

**Comment by mli** [comment]
Job d2l-zh/PR-615/4 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-615/

**Comment by mli** [comment]
Job d2l-zh/PR-615/5 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-615/

**Comment by mli** [comment]
Job d2l-zh/PR-615/6 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-615/

**Comment by mli** [comment]
Job d2l-zh/PR-615/7 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-615/

**Pull request merged**